docker-compose.yaml

```diff
@@ -51,11 +51,12 @@ services:
                 condition: service_healthy
 
     database:
-        image: postgres:15.2-alpine3.17
+        image: postgres:15.6-alpine3.19
         container_name: escooters-db-dev
         environment:
             - POSTGRES_USER=${DB_USERNAME}
-            - POSTGRES_PASSWORD=${DB_PASSWORD}
+            - POSTGRES_PASSWORD=${DB_PASSWORD}FROM node:21.7.1-bullseye-slim as node
```

Comment:
There seems to be a mistake here - we don't need here _FROM node:21.7.1-bullseye-slim as node_

Reply:
fixed